crates/comments/src/directive.rs

```diff
@@ -30,15 +30,21 @@ pub enum FormatDirective {
 ///
 /// Note that directives are applied to the node they are attached to.
 pub fn parse_comment_directive(text: &str) -> Option<Directive> {
-    let text = text.strip_prefix('#')?;
+    let text = text.trim_start_matches('#');
```

Comment:
```suggestion
    let text = text.strip_prefix('#')?;
    let text = text.trim_start_matches('#');
```

I propose this to enforce that at least 1 `#` must be present in `text`

Reply:
I've also added a comment with the example usage you shared with me yesterday.